ttyplot.c

```diff
@@ -91,7 +91,9 @@ static void usage(void) {
             "  -t title of the plot\n"
             "  -u unit displayed beside vertical bar\n"
             "  -v print the current version and exit\n"
-            "  -h print this help message and exit\n\n");
+            "  -h print this help message and exit\n"
+            "\n"
+            "Interactive commands: q = quit, r = toggle rate mode\n");
```

Comment:
My vote for replacing "commands" by e.g. "hotkeys" and for making this last line three lines, for readability.

PS: The help output dump in the readme is no longer in sync with reality after this. Would be great to have in sync.

Reply:
OK, replaced with “hotkeys”. Note that the dump of the help output in the README is not a complete dump, as it lacks the line “Usage:”. Thus I used a dump of the second part of the help in the following section.

Reply:
> OK, replaced with “hotkeys”.

@edgar-bonet thanks!

> Note that the dump of the help output in the README is not a complete dump, as it lacks the line “Usage:”. Thus I used a dump of the second part of the help in the following section.

@edgar-bonet I think midterm ideally it would be an identically copy. I have CI elsewhere to keep help output copies in sync…

- https://github.com/git-big-picture/git-big-picture/blob/main/.github/workflows/enforce-copies-of-help-output-in-sync.yml
- https://github.com/hartwork/git-delete-merged-branches/blob/master/.github/workflows/enforce-copies-of-help-output-in-sync.yml

…so something like that would be an option. It prevents cases like https://github.com/hartwork/resolve-march-native/pull/113. Midterm, just an idea, felt like mentioning it, a great case for something that CI can do in my opinion.